cluster/images/conformance/Makefile

```diff
@@ -31,7 +31,7 @@ E2E_GO_RUNNER_BIN?=$(shell test -f $(LOCAL_OUTPUT_PATH)/go-runner && echo $(LOCA
 
 CLUSTER_DIR?=$(shell pwd)/../../../cluster/
 
-BASEIMAGE=debian:stable-slim
+BASEIMAGE=k8s.gcr.io/debian-base:buster-v1.4.0
```

Comment:
TODO: should probably respect KUBE_BASE_IMAGE_REGISTRY

I'm more interested in eliminating the dockerhub dependency first though.